polyfill/lib/duration.mjs

```diff
@@ -130,6 +130,9 @@ export class Duration {
     if (!dateParts.length && !timeParts.length) return 'PT0S';
     return `P${dateParts.join('')}${timeParts.join('')}`;
   }
+  toLocaleString(...args) {
+    return new Intl.DateTimeFormat(...args).format(this);
```

Comment:
shouldn’t this brand check the receiver first?

Reply:
@pipobscure ?

Reply:
#224